app/src/main/java/com/asafvaron/themoviedbtest/fragments/MovieInfoFragment.java

```diff
@@ -0,0 +1,54 @@
+package com.asafvaron.themoviedbtest.fragments;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.asafvaron.themoviedbtest.R;
+import com.asafvaron.themoviedbtest.model.Movie;
+import com.bumptech.glide.Glide;
+
+/**
+ * Created by asafvaron on 20/02/2017.
+ */
+public class MovieInfoFragment extends Fragment {
+    private static final String TAG = MovieInfoFragment.class.getSimpleName();
+    private Movie mMovie;
+
+    public static MovieInfoFragment newInstance(Movie movie) {
+        MovieInfoFragment fragment = new MovieInfoFragment();
+        Bundle args = new Bundle();
+        args.putSerializable("movie", movie);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments()!=null)
+            mMovie = (Movie) getArguments().getSerializable("movie");
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View root = inflater.inflate(R.layout.movie_info_layout, container, false);
+
+        Glide.with(getActivity())
```

Comment:
Check for reusing the url from the movie grid view so it won't need to reload from network and you might be able to have a nice transition later on...